shakedown/dcos/package.py

```diff
@@ -91,8 +91,6 @@ def install_package(
         labels = pkg.marathon_json(options).get('labels')
         if 'DCOS_SERVICE_NAME' in labels:
             service_name = labels['DCOS_SERVICE_NAME']
```

Comment:
Hm, what happens when `DCOS_SERVICE_NAME` is not in labels? Will `service_name` be `NONE`?